Resources/Prototypes/Entities/Stations/base.yml

```diff
@@ -71,3 +71,9 @@
   abstract: true
   components:
     - type: StationEventEligible # For when someone makes this more granular in the future.
+
+- type: entity
+  id: BaseStationRenameFaxes
+  abstract: true
+  components:
+    - type: StationRenameFaxes
```

Comment:
could you stuff this into uh the shipyard or something instead? then this looks good to go and yea, should be an easy text edit to update the ships themselves

Reply:
@Cheackraze I can move this to shipyard console, but it won't handle if station/ship would spawn some other way (maps, admeme, etc). Also this won't work if ship change name. This component can be attached to any station proto.

Reply:
yes but put it in the new frontiers shipyard catalog file instead, not in the base.yml file